Add documentation for Qiscus channel (#95)
kata-ai/kata-platform-docs: docs/channels/qiscus.md

---
id: channel-qiscus
title: Qiscus
---

Qiscus is an AI compliant, secure and flexible chat SDK for apps and websites. They also provide multichannel customer service chat solution which is now getting popular as it is best designed for everyone to enhance their customer service experiences using multiple channels and bot-human collaboration customer service. In this section, we will learn on how to connect our bot with Qiscus thorough these following steps.

## Create Qiscus Channel

> 1. First thing first, you need to register yourself at [Qiscus Multichannel CS Chat Website](https://multichannel.qiscus.com) to create a new app. You will be asked to fill in several information regarding your account.

Reviewer: `First thing first` agak kurang general menurutku, ada alternative kata lain gak ya kira kira?
tapi ini ga harus diganti si hahaha

Author: menurutku udah oke sih first thing first, gakepikiran yg lain

> 1. First thing first, you need to register yourself at [Qiscus Multichannel CS Chat Website](https://multichannel.qiscus.com) to create a new app. You will be asked to fill in several information regarding your account.

Reviewer: ```suggestion
1. First thing first, you need to register yourself at [Qiscus Multichannel CS Chat Website](https://multichannel.qiscus.com) to create a new app. You will be asked to complete the given form.
```

Author: aku lebih suka kalimatku tapi wkwk

![Register](./images/qiscus/sensor_register.jpg)

> 2. As soon as you signed up, you will get an email verification from Qiscus Multichannel CS Chat that you need to verify before you can use it. So just check your email and verify it.

Reviewer: ```suggestion
2. As soon as you signed up, verify your account by click on the link from Qiscus Multichannel CS Chat email. Only verified accounts that able to use **Qiscus**.
```

Author: ini juga aku lebih suka kalimatku hehe

![Verify](./images/qiscus/validate.png)

> 3. After you verified your account, you can log into the Qiscus Multichannel CS Chat dashboard via [login page](https://multichannel.qiscus.com). Please fill in your credentials to get in to the dashboard.

Reviewer 2: point 3. in to --> into

4. Once you logged in, go to the integration page, and enable the "Qiscus Widget" tab like shown on picture below.
![Qiscus Widget Integration](./images/qiscus/red_qiscuswelcomepage.png)

> 5. Copy the snippet code shown on the page and paste it into your website html. It is not set yet, you can open your html later after you integrate the bot.

Reviewer: ```suggestion
5. Copy the snippet code shown on the page and paste it into your _**website html**_. It is not set yet, you can open your html later after you integrate the bot.
```

`It is not set yet,` itu mksdnya gimana?

Author: maksudnya belom selesai gitu prosesnya, bagusnya kalimatnya gimana yak

Reviewer: `you will use your html code later blabla`

> 6. As predicted, the next step is you need to integrate the bot by enabling the "Bot Integration" in the Integration Page. You will also see Agent ID, App ID, Qiscus Secret Key, URL. These four things will be needed later in defining new channel in Kata Platform. There's also one blank space in "Bot Webhook URL" which can be filled later if you've created the channel in Kata Platform.

Reviewer: ```suggestion
6. As predicted, the next step is you need to integrate the bot by enabling the "Bot Integration" in the Integration Page. You will also see Agent ID, App ID, Qiscus Secret Key, URL. These four things will be needed later in defining a new channel in Kata Platform. There's also one blank space in "Bot Webhook URL" which can be filled later after creating the channel in Kata Platform.
```

Author: oke yg ini aku setuju kelimatmu. will fix it asap. thanku

![Bot Integration](./images/qiscus/sensor_botintegration.jpg)

## Create Qiscus Channel in Kata Platform

> First, login to Kata Platform and select the project that you want to integrate with Qiscus. Then, deploy the project by selecting "Deploy" > "New Deployment".

Reviewer: ```suggestion
First, login to Kata Platform and select the project that you want to integrate with Qiscus (or create if you don't have one). Then, deploy the project by selecting "Deploy" > "New Deployment".
```

Author: oke noted. thanku

![Create Environment in Kata Platform](./images/qiscus/red_environmenttrue.png)

Fill the "Environment URL" and click "Create".
![Create Environment in Kata Platform](./images/qiscus/sensor_createenvironment.jpg)

Now, you should create the channel by selecting "Create Channel".
![Create Channel in Kata Platform](./images/qiscus/red_createchannel.jpg)

You will now see the "Create Channel" screen. Insert the channel name, and select Qiscus as the channel type.
![Create Channel in Kata Platform](./images/qiscus/red_createchannel2.png)
![Create Channel in Kata Platform](./images/qiscus/red_createchannel3.png)

Then, copy and paste the previously Secret Key, Bot Email, and URL, and click "Create".
![Create Channel in Kata Platform](./images/qiscus/red_createchannel4.png)

## Setup Webhook URL in Qiscus Bot

Once finished, you will see the newly-created channel in the list. You will also see a Webhook URL for your channel. This webhook URL will be added to the Qiscus bot. Copy the webhook URL by clicking the copy button next to the Webhook URL field.
![Webhook URL in Kata Platform](./images/qiscus/red_webhook.png)

> Next, back to Qiscus dashboard, and go to "Bot Integration" tab. Make sure you've enabled it, then paste the webhook URL in "Bot Webhook URL", and click "Connect".

Reviewer: ```suggestion
Next, head back to Qiscus dashboard, and go to the "Bot Integration" tab. Make sure you've enabled it, then paste the webhook URL in "Bot Webhook URL", and click "Connect".
```

Author: oke noted. thankuu

![Webhook URL in Qiscus](./images/qiscus/red_webhook2.png)

Now, you can check your Qiscus bot in the Qiscus widget by accessing your previous html file. And, voila! It works! You can now share your newly created chatbot with your friends.

> ![Qiscus Bot in Qiscus Widget](./images/qiscus/result.png)

Reviewer: Gambarnya di crop terus di taruh di center gitu aja bel biar rapih

Author: okayy
